src/components/FlowRunWorkQueue.vue

```diff
@@ -18,6 +19,7 @@
   }>()
 
   const isNotTerminal = computed(() => props.flowRunState && !isTerminalStateType(props.flowRunState))
+  const { workPool } = useWorkPool(props.workPoolName ?? '')
```

Comment:
Looking at this closer this probably works but might not work quite as expected. You'll probably want to do

```suggestion
  const { workPoolName } = toRefs(props)
  const { workPool } = useWorkPool(workPoolName)
```

Which will retain reactivity even if the prop starts as undefined but changes.